supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,107 @@
+** Avoid indicating direction, such as left, right, above and below, since these have no meaning to a screen reader. Instructions provided for understanding and operating the product must not rely solely on sensory characteristics such as shape, size, visual location, or orientation.
+
+* Avoid instructions that rely solely on sensory characteristics.
+* Information that is conveyed by color differences must also be provided in text form.
+* For images or diagrams in your document, ensure sufficient contrast between foreground and background text or graphical elements. Graphics and diagrams in your document produced by a CCS Designer meet contrast ratio requirements. If you created a graphic or diagram yourself, use the WCAG reference for link:https://www.w3.org/TR/WCAG21/#contrast-minimum[Contrast], and the link:https://webaim.org/resources/contrastchecker/[Contrast Checker], to evaluate contrast compliance.
+* Avoid directional indicators such as left, right, above and below when providing navigational instruction in the user interface.
+
+=== Well-formed HTML and Meaningful Sequence
+
+*Accessibility criteria:* The meaning of content relies on the order in which you present it. For example, English content is arranged from left to right and readers usually look at a left-hand column before a right-hand column. Users who rely on assistive technology (such as a screen reader) to interpret content need the content to be presented in a meaningful order. If content is presented out of sequence, readers may become disoriented and unable to understand the content.
+
+* Correctly nest headings from H1 to H2, H3 and so forth without skipping heading levels to indicate relative importance.
+* Use correct Asciidoc tags to produce intended HTML.
+
+
+.Reference
+
+*  Understanding WCAG 2.0
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv.html[Text Alternatives].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-text-presentation.html[Images of Text].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html[Link Purpose(In Context)].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-understanding.html[Sensory Characteristics].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html[Use of Color].
+** link:https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html[Meaningful Sequence]
+* link:https://webaim.org/resources/contrastchecker/[WebAIM Contrast Checker].
+
+== Localization
+
+Documentation that follows the IBM Style Guide for fairly conversational tone does not present issues with human- or machine-translation. Use of contractions, specifically, has not shown to produce issues with translations.
```

Comment:
* IBM Style Guide in italics
* I don't think you need the dashes, so I'd just do "human or machine translation"
* "Use of" -> "The use of" . Or actually I might consider making it more direct in giving guidance that you can use contractions.

Reply:
Should we mention anything about how using 'fairly conversational tone' can better align the tone of the docs with the tone of the UI? Also OK to keep it vague as just an option here too, up for interpretation by the writer, if the connection seems to already be clear.

Reply:
@dayleparker @bergerhoffer I revised this section based on feedback provided by each of you. Please lemme know what you think.